<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1187</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Tableau">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>1101</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Gestion des promotions</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>1061</width>
        <height>501</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Ajouter une promotion</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_reference">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>150</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_pourcentage">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>200</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>150</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Reference du produit</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>200</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Pourcentage</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>250</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>date de debut</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>300</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Dernier délai </string>
        </property>
       </widget>
       <widget class="QPushButton" name="Ajouter_Button">
        <property name="geometry">
         <rect>
          <x>830</x>
          <y>400</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_debut">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>250</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_fin">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>300</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Supprimer une promotion</string>
       </attribute>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>80</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>La reference</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_supp">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>80</y>
          <width>231</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="Supp_Button">
        <property name="geometry">
         <rect>
          <x>720</x>
          <y>260</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Valider</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Afficher les promotions</string>
       </attribute>
       <widget class="QTableView" name="tabpromotions">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>40</y>
          <width>691</width>
          <height>321</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Modifier une promotion</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_reference_2">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>80</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>180</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>date de debut</string>
        </property>
       </widget>
       <widget class="QDateTimeEdit" name="dateTimeEdit_fin_2">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>230</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QDialogButtonBox" name="buttonBox_5">
        <property name="geometry">
         <rect>
          <x>790</x>
          <y>360</y>
          <width>193</width>
          <height>28</height>
         </rect>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>230</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Dernier délai </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_pourcentage_2">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>130</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>80</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Reference du produit</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>130</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Pourcentage</string>
        </property>
       </widget>
       <widget class="QDateTimeEdit" name="dateTimeEdit_debut_2">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>180</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Chercher une promotion</string>
       </attribute>
       <widget class="QDialogButtonBox" name="buttonBox_6">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>280</y>
          <width>193</width>
          <height>28</height>
         </rect>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>130</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>La reference</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_supp_2">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>130</y>
          <width>231</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_2">
     <attribute name="title">
      <string>Gestion</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1187</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
